### PR TITLE
[atapp@linuxhost atapp@mk3060]

### PR DESCRIPTION
**Pull request by hptsf**
- fix bug: build failed for the two projects(missing header files)

### CODE REVIEW

File: board/linuxhost/linuxhost.mk [hunk 1]
```diff
@@ -30,7 +30,7 @@ GLOBAL_ASMFLAGS += -m32
 GLOBAL_CFLAGS += -m32  -std=gnu99
 GLOBAL_LDFLAGS += -m32
 
-GLOBAL_INCLUDES += .
+GLOBAL_INCLUDES += . ../../device/sal/wifi/mk3060
```
Comment (junjiec) [inline board/linuxhost/linuxhost.mk]: 编译的时候，指定sal=1，会把device/sal/wifi/mk3060加入编译，device/sal/wifi/mk3060会被加到搜索路径：
aos make atapp@linuxhost sal=1

File: board/mk3060/mk3060.mk [hunk 2]
```diff
@@ -10,7 +10,7 @@ SUPPORT_BINS         := no
 
 $(NAME)_SOURCES := board.c
 
-GLOBAL_INCLUDES += .
+GLOBAL_INCLUDES += . ../../device/sal/wifi/mk3060
```
Comment (junjiec) [inline board/mk3060/mk3060.mk]: 同上，参考linuxhost，通过sal环境变量来选择编译

File: board/linuxhost/linuxhost.mk (hunk 1 above)
Comment (hptsf) [reply board/linuxhost/linuxhost.mk]: image
Thanks for your reply.
But, why build failed on my pc(ubuntu-14.04)? Any config files missing?

File: board/mk3060/mk3060.mk (hunk 2 above)
Comment (hptsf) [reply board/mk3060/mk3060.mk]: image

As pic, i type grep command in Alios-things and aos installed dir, there haven't any if/else logic for "sal=1" or "sal=0".

Comment (junjiec) [reply board/mk3060/mk3060.mk]: board/linuxhost/linuxhost.mk
 37 ifeq (1,$(sal))
 38 $(NAME)_COMPONENTS += sal
 39 module ?= wifi.mk3060
 40 endif

Comment (hptsf) [reply board/mk3060/mk3060.mk]: image
image

There haven't mk3060?

Comment (junjiec) [reply board/mk3060/mk3060.mk]: mk3060本身就是WiFi SoC，所以不需要
linuxhost或者starterkit支持

Comment (hptsf) [reply board/mk3060/mk3060.mk]: when i want to build mk3060, whether or not i add "sal=1", it will build failed.
Maybe you can re-try on your pc.

**Comment by skylarCai** [comment]
感谢您提出的宝贵问题，我们有7x24小时服务的“HaaS百事通”客服系统，
        您可以先尝试能否解决您的问题(https://haas.iot.aliyun.com/?ask=1&f=a2cre.b82925042)
        如果“HaaS百事通”没有解决您的问题，我们会在两个工作日回复您。

**Pull request closed without merging**